deploy/helm/grafana-operator/templates/dashboard.yaml

```
@@ -0,0 +1,20 @@
+{{- if .Values.dashboard.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "grafana-operator.fullname" . }}-dashboard
+  namespace: {{ include "grafana-operator.namespace" . }}
+  labels:
+    {{- include "grafana-operator.labels" . | nindent 4 }}
+    app.kubernetes.io/component: operator
+    {{- with .Values.dashboard.additionalLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  {{- with .Values.dashboard.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+data:
+  grafana-operator: |-
```

I think file provisioner will not pick up the dashboard unless the file contains correct extension.

```suggestion
  grafana-operator.json: |-
```